lr-cli/cortx_setup/commands/node/finalize.py

```diff
@@ -215,6 +215,29 @@ def _create_field_users(self, node_id):
                     targets=node_id
                 )
 
+            else:
+                self.logger.debug(f"Ensuring {user} user permissions on system")
+
+                StateFunExecuter.execute(
+                    'file.append',
+                    fun_kwargs=dict(
+                        name=f'{home_dir}/.bashrc',
+                        text='alias nodecli="sudo nodecli"'
```

Comment:
Change this to:
`alias nodecli="sudo /usr/bin/nodecli"`

Reply:
done!